install/cupy_builder/install_build.py

```diff
@@ -684,14 +684,23 @@ def conda_get_target_name():
     return out
 
 
+@functools.cache
+def is_conda_cross_compiling() -> bool:
+    is_cross_compiling = os.environ.get('CONDA_BUILD_CROSS_COMPILATION', '0')
+    # Recent conda compilers might set CONDA_BUILD_CROSS_COMPILATION to empty.
+    if is_cross_compiling == '':
+        is_cross_compiling = '0'
```

Comment:
This is the fix (to handle the case when the env var is set to an empty string).